modules/configuring-private-storage-endpoint-azure-user-provided-vnet-subnet.adoc

```diff
@@ -0,0 +1,102 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/configuring-private-cluster.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="configuring-private-storage-endpoint-azure-user-provided-vnet-subnet"]
+= Configuring a private storage endpoint on Azure with user-provided VNet and subnet names
+
+Use the following procedure to configure a private storage endpoint on Azure with user-provided VNet and subnet names.
```

Comment:
Private storage endpoint is not a thing in azure. I'm okay with introducing this term here as it does make sense. The terms used in azure are "private endpoint" and "storage account". In the code I've been using "private storage account" to refer to a storage account that has public network access disabled and is exposed behind a private endpoint.
I'll leave it to you to decide how to proceed.